README.md

```diff
@@ -66,11 +70,22 @@ $ tensorboard --logdir tensorboard_log/
 ```
 ## Rolling out a trained agent
 
-To rollout a trained agent that is stored at 'complex_envs/my_first_agent.pkl' for 9 steps, run the following command from the same folder:
+To rollout a trained agent that is stored at `complex_envs/my_first_agent.pkl` for 60 steps, run the following command from the same folder:
 
 ```
-$ python rollout.py -n 9 -i my_first_agent.pkl -e VibratingBridge
+$ python rollout.py -n 60 -i my_first_agent.pkl
 ```
 
-The rollout will be saved in a file format starting with 'output', which can be changed by passing `-f <filename>` to the above command.
+The rollout will be saved as `output.npz`, which can be changed by passing `-f <filename>` to the above command.  Note that the trajectories produced will have length equal to the number of rollout steps + the number of warmup steps + the number of equilibriation steps (values of which are set in `/configs/config.yml`).
+
+### Visualizing a rollout
+
+To visualize a rollout saved in `complex_envs/output.npz`, run the following command from the same folder:
+
+```
+# python visualize.py -i output.npz -f first_output
+```
+
+This will produce two files `first_output.eps` which plots the trajectory of the energy over the episode and `first_output.gif` which is an animation of the bridge and the impulse force.
```

Comment:
.gif file is a series of images that must be scrolled through, not a gif

Reply:
Does it actually produce a .gif file?  This code produces a properly animated file on my computer and a fellow's mac.  Can you try opening it with multiple programs?

Reply:
I'll add a comment to make sure to check what program to open it in.